src/test/java/it/gov/pagopa/payhub/activities/connector/organization/client/BrokerClientTest.java

```diff
@@ -49,4 +50,24 @@ void testFetchAll() {
 		verify(organizationApisHolderMock.getBrokerEntityControllerApi(accessToken), times(1))
 			.crudGetBrokers(DEFAULT_PAGE_NUMBER, DEFAULT_PAGE_SIZE, null);
 	}
+
+	@Test
+	void testGetByFiscalCode() {
+		// Given
+		String fiscalCode = "ABCDEF12G34H567I";
+		String accessToken = "accessToken";
+		Broker expectedBroker = mock(Broker.class);
+		var brokerSearchControllerApi = mock(it.gov.pagopa.pu.organization.client.generated.BrokerSearchControllerApi.class);
+
+		when(organizationApisHolderMock.getBrokerSearchControllerApi(accessToken)).thenReturn(brokerSearchControllerApi);
+		when(brokerSearchControllerApi.crudBrokersFindByBrokeredOrgFiscalCode(fiscalCode)).thenReturn(expectedBroker);
+
+		// When
+		Broker result = client.getByFiscalCode(fiscalCode, accessToken);
+
+		// Then
+		assertEquals(expectedBroker, result);
+		verify(organizationApisHolderMock).getBrokerSearchControllerApi(accessToken);
+		verify(brokerSearchControllerApi).crudBrokersFindByBrokeredOrgFiscalCode(fiscalCode);
```

Comment:
If you configure a Mockito.when invocation configuring its parameter, the verify invocation is superfluous when you expect just one invocation.

Checking AfterEach test noMoreInvocations on mocks will complete this check: thus you could remove these 2 lines